setup.cfg

```diff
@@ -112,6 +112,12 @@ where = src
 [options.entry_points]
 console_scripts =
     briefcase = briefcase.__main__:main
+briefcase.bootstraps =
+    Toga = briefcase.bootstraps.toga:TogaGuiBootstrap
+    PySide2 = briefcase.bootstraps.pyside2:PySide2GuiBootstrap
+    PySide6 = briefcase.bootstraps.pyside6:PySide6GuiBootstrap
+    PursuedPyBear = briefcase.bootstraps.pursuedpybear:PursuedPyBearGuiBootstrap
+    Pygame = briefcase.bootstraps.pygame:PygameGuiBootstrap
```

Comment:
I ultimately removed the class attribute `name` from the bootstrap class definition. That means that the name defined here for the entry point is also used for display purposes. In general, I'd expect the entry point name and the GUI display name to be the same....and that makes keeping track of them easier....but we can separate the concepts if that's a better design choice.

Reply:
The place where I see this breaking is when the bootstrap isn't *just* a GUI framework - e.g., a Positron project with a Django backend. A guess a name like "Positron-Django" could work, but having the flexibility to say "Positron (Django server)" would be nice.

Reply:
Can you clarify what you mean exactly?

If you're concerned about more interestingly named entry points....that doesn't seem to be to problem for me.

For example:
```
briefcase.bootstraps =
    Toga = briefcase.bootstraps.toga:TogaGuiBootstrap
    PySide2 = briefcase.bootstraps.pyside2:PySide2GuiBootstrap
    PySide6 = briefcase.bootstraps.pyside6:PySide6GuiBootstrap
    PursuedPyBear = briefcase.bootstraps.pursuedpybear:PursuedPyBearGuiBootstrap
    Pygame = briefcase.bootstraps.pygame:PygameGuiBootstrap
    Pygame (Testing) = briefcase.bootstraps.pygame:PygameGuiBootstrap
```
```
What GUI toolkit do you want to use for this project?

  1) Toga
  2) PySide2                  (does not support iOS/Android deployment)
  3) PySide6                  (does not support iOS/Android deployment)
  4) PursuedPyBear            (does not support iOS/Android deployment)
  5) Pygame                   (does not support iOS/Android deployment)
  6) PursuedPyBear Automation (does not support iOS/Android deployment)
  7) PySide2 Automation       (does not support iOS/Android deployment)
  8) PySide6 Automation       (does not support iOS/Android deployment)
  9) Pygame (Testing)         (does not support iOS/Android deployment)
  10) Pygame Automation        (does not support iOS/Android deployment)
  11) Toga Automation
  12) None
```
Note that I also have a plugin installed with a bunch of entry points.

Reply:
Huh. TIL. I thought plugin names needed to be Python identifiers.